source/index.txt

```diff
@@ -28,20 +28,24 @@ presents a challenge when working with MongoDB's
 :manual:`flexible schema </data-modeling>` and rich,
 multi-dimensional documents.
 
-The |bi| acts as a layer that passes queries and
-data between a :program:`mongod` or :program:`mongos` instance and your
-reporting tool. The |bi| stores no data, and purely serves to bridge your
-MongoDB server with business intelligence tools.
+The |bi| acts as a layer that passes queries and data between a
+:program:`mongod` or :program:`mongos` instance and your reporting tool.
+The |bi| stores no data, and purely serves to bridge your MongoDB server
```

Comment:
/s/server/cluster ? A server implies a single server, e.g. a standalone. Moreso given a sharded cluster.

Reply:
Fixed.

Reply:
I think the change didn't get pushed?